homeassistant/components/homee/sensor.py

```diff
@@ -149,15 +149,20 @@ class HomeeSensorEntityDescription(SensorEntityDescription):
         key="position",
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    AttributeType.RAINFALL_INTENSITY: HomeeSensorEntityDescription(
+        key="precipitation_intensity",
+        device_class=SensorDeviceClass.PRECIPITATION_INTENSITY,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
```

Comment:
This PR is doing multiple things. Adding new sensors and applying fixes for others.

This needs to be split out into separate pull requests.

../Frenck